docs/src/apireference.md

```diff
@@ -96,47 +96,18 @@ The value of the attribute is of type `ResultStatusCode`.
 ResultStatusCode
 ```
 
+## Variables and Constraints
+
 ### Basis Status
 
-[what? BasisStatus]
+The `BasisStatus` attribute of a variable or constraint indicates its presence in a basis.
```

Comment:
There's more than one way in which a variable/constraint may be *not* in the basis (i.e., at an upper or lower bound), so this wording is a bit unclear.